rfcs/RFC-0004-proof-of-relay/0004-proof-of-relay.md

```
@@ -2,44 +2,139 @@
+in [IETF RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+### 2.1 Cryptographic and security parameters
+
+The documents make use of certain cryptographic and mathematical terms. A security parameter `L` is chosen
+and corresponding cryptographic primitives are used in a concrete instantiations of this RFC.
+The speficic instantiation of the current version of this protocol is given in Appendix 1.
+
+The security parameter `L` SHALL NOT be less than 2^128 - meaning the chosen cryptographic primitives
+instantiations below SHALL NOT have less than 128-bits of security.
+
+- **EC group** refers to a specific elliptic curve group over a finite field, where computational Diffie-Hellman problem is AT LEAST as
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 120; Actual: 135

(MD013, line-length)